chromadb/server/fastapi/__init__.py

```diff
@@ -679,6 +695,7 @@ async def get_collection(
         database_name: str,
         collection_name: str,
     ) -> CollectionModel:
+        add_attributes_to_current_span({"pod_name": pod_name()})
```

Comment:
Can this be folded into the `trace_method` decorator?

Reply:
Good idea, this is useful in any trace